parser_2gis/config.py

```diff
@@ -21,13 +21,13 @@ class Configuration(BaseModel):
     writer: WriterOptions = WriterOptions()
     chrome: ChromeOptions = ChromeOptions()
     parser: ParserOptions = ParserOptions()
-    path: Optional[pathlib.Path]
+    path: Optional[pathlib.Path] = None
     version: str = config_version
 
     def __init__(self, *args, **kwargs) -> None:
         def setup_config(model: BaseModel) -> None:
             """Recursively setup config."""
-            self.Config.validate_assignment = True
+            # self.Config.validate_assignment = True
```

Comment:
Assignments validation is important for any settings changes, why have you turned it off?